notebooks/003-hello-segmentation/003-hello-segmentation.ipynb

```diff
@@ -149,10 +145,10 @@
    "execution_count": null,
    "source": [
     "# Define titles with images\n",
-    "data = {'Segmentation': mask, \"Base Photo\": input_image_t, \"Masked Photo\": image_with_mask}\n",
+    "data = {'Segmentation': mask, \"Base Photo\": resized_image, \"Masked Photo\": image_with_mask}\n",
```

Comment:
Can you switch around the base photo and the segmentation so the photo is left, then the mask, then the image?